core/src/test/resources/application_test.conf

```diff
@@ -26,6 +26,9 @@
 
     # Set the free memory requirement much lower for running tests
     min-free-mb = 10
+
+    filo.chunksize = 100
+    flush.interval = 5 s    # Make this high enough that tests will require manual flushing usually
```

Comment:
May be easier to read this way, and it's how the akka confs are written, just a thought.

     # Make this high enough that tests will require manual flushing usually
     flush.interval = 5 s